great_expectations/execution_engine/execution_engine.py

```diff
@@ -479,12 +480,18 @@ def _build_direct_and_bundled_metric_computation_configurations(
         Returns:
             Tuple with two elements: directly-computable and bundled "MetricComputationConfiguration" objects
         """
-        if metrics is None:
-            metrics = {}
-
         metric_fn_direct_configurations: List[MetricComputationConfiguration] = []
         metric_fn_bundle_configurations: List[MetricComputationConfiguration] = []
 
+        if not metrics_to_resolve:
+            return (
+                metric_fn_direct_configurations,
+                metric_fn_bundle_configurations,
+            )
+
+        if metrics is None:
```

Comment:
Probably want to keep with the pattern used elsewhere: `if not metrics`

Reply:
@NathanFarmer In this case we are actually focusing on "None" (as opposed to "having nothing in it" -- i.e., being "Falsy").  So this is a stronger statement.  Happy to discuss.  Thanks!